### PR TITLE
CLB 19/06

### PR DESCRIPTION
**Pull request by JohnWilliams77**

### CODE REVIEW

**Review by Agetian — DISMISSED** [review]

File: forge-gui/res/cardsfolder/upcoming/armor_of_shadows.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Name:Armor of Shadows
+ManaCost:B
+Types:Instant
+A:SP$ Pump | Cost$ B | ValidTgts$ Creature | AILogic$ Pump | NumAtt$ +1 | KW$ Indestructible | SpellDescription$ Until end of turn, target creature gets +1/+0 and gains indestructible.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/armor_of_shadows.txt:4]: Cost$ is uneeded here

File: forge-gui/res/cardsfolder/upcoming/arms_of_hadar.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Name:Arms of Hadar
+ManaCost:3 B
+Types:Sorcery
+A:SP$ PumpAll | Cost$ 3 B | ValidTgts$ Player | TgtPrompt$ Select target player | ValidCards$ Creature | NumAtt$ -2 | NumDef$ -2 | SpellDescription$ Creatures target player controls get -2/-2 until end of turn.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/arms_of_hadar.txt:4]: Cost$ is uneeded here

File: forge-gui/res/cardsfolder/upcoming/atrocious_experiment.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Atrocious Experiment
+ManaCost:2 B
+Types:Sorcery
+A:SP$ Mill | Cost$ 2 B | NumCards$ 2 | ValidTgts$ Player | TgtPrompt$ Choose a player | SubAbility$ DBDraw | SpellDescription$ Target player mills two cards, draws two cards, and loses 2 life.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/atrocious_experiment.txt:4]: Cost$ is uneeded here

File: forge-gui/res/cardsfolder/upcoming/armor_of_shadows.txt (hunk 1 above)
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/armor_of_shadows.txt:4]: SVar:PlayMain1:TRUE

File: forge-gui/res/cardsfolder/upcoming/atrocious_experiment.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Atrocious Experiment
+ManaCost:2 B
+Types:Sorcery
+A:SP$ Mill | NumCards$ 2 | ValidTgts$ Player | TgtPrompt$ Choose a player | SubAbility$ DBDraw | SpellDescription$ Target player mills two cards, draws two cards, and loses 2 life.
+SVar:DBDraw:DB$ Draw | NumCards$ 2 | SubAbility$ DBLoseLife
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/atrocious_experiment.txt:5]: need `Defined$ Targeted`

**Review by Hanmac — APPROVED** [review]
👍

**Comment by Hanmac** [comment]
@Agetian the "Mill 2, Draw 2, Lose 2 Life" probably needs some more AI logic for their effects
so AI can better decide it it wants to target itself or opponent with it? some cards that does "Draw 2, lose 2 Life" already have some logic, but i don't know if that also works if Mill is first.

**Comment by Agetian** [comment]
Yeah, I might test some corner cases when the number of cards in the library is low for those cards, hopefully July will present some opportunities for things like that - adding to my todo list ;)

File: forge-gui/res/cardsfolder/upcoming/atrocious_experiment.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Atrocious Experiment
+ManaCost:2 B
+Types:Sorcery
+A:SP$ Mill | NumCards$ 2 | ValidTgts$ Player | TgtPrompt$ Choose a player | SubAbility$ DBDraw | SpellDescription$ Target player mills two cards, draws two cards, and loses 2 life.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/atrocious_experiment.txt:4]: Probably just remove `TgtPrompt$ Choose a player | `

Forge can parse simple stuff like this no problem

File: forge-gui/res/cardsfolder/upcoming/arms_of_hadar.txt [hunk 6]
```diff
@@ -0,0 +1,5 @@
+Name:Arms of Hadar
+ManaCost:3 B
+Types:Sorcery
+A:SP$ PumpAll | ValidTgts$ Player | TgtPrompt$ Select target player | ValidCards$ Creature | NumAtt$ -2 | NumDef$ -2 | SpellDescription$ Creatures target player controls get -2/-2 until end of turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/arms_of_hadar.txt:4]: You can cut TgtPrompt here too if you like
